joomla/joomla-cms
# [4.0] Some modal fixes

Pull Request for Issue #18605 .

### Summary of Changes
Fixes styling of the modal close (top-right). Adds vertical overflow to modal body


### Testing Instructions
Apply PR and navigate to Extensions -> Templates -> Templates -> New File. Ensure modal is now displaying correctly.


### Before
![image](https://user-images.githubusercontent.com/2803503/32981763-ee0a233a-cc71-11e7-987d-36db7deda6bd.png)

### After
![image](https://user-images.githubusercontent.com/2803503/32981800-28421882-cc72-11e7-85aa-dd407276d58b.png)



### Documentation Changes Required
